tasks/copy_to_glacier/handler.py

```diff
@@ -115,42 +83,37 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
             input (list): event['input']
     """
     print(event)
-    granule_urls = event.get('input', [])
+    event_input = event.get('input')
+    # If there is no granules object, fail the workflow.
+    granules_list = event_input.get('granules')
```

Comment:
This code will actually run just fine even if 'granules' is missing. granules_list will be None.

Reply:
Good catch. I will remove that comment.

Reply:
Is the case where granules_list == None handled?

Reply:
Doesn't look like there's a unit test around it either. I feel like that's a raiseable offense, but we could also consider it equal to granules_list being empty.